motrainer/util.py

```diff
@@ -3,7 +3,7 @@
 import os
 import sklearn.preprocessing
 import random
-from scipy.stats.stats import pearsonr, spearmanr
+from scipy.stats import pearsonr, spearmanr
 
 os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'  # Force tensorflow debug logging off
```

Comment:
This file is also changed in #97, can you please merge the generalization into your branch? There, I changed the value of `os.environ['TF_CPP_MIN_LOG_LEVEL']` from '3' to '2'. :smile: